docs/core/diagnostics/cli-tools/dotnet-trace.md

```diff
@@ -0,0 +1,151 @@
+
+collect:
+  Collects a diagnostic trace from a currently running process
+
+Usage:
+  dotnet-trace collect [options]
+
+Options:
+  -h, --help
+    Shows this help message and exit.
+
+  -p, --process-id <pid>
+    The process to collect the trace from
+
+  -o, --output <trace-file-path>
+    The output path for the collected trace data. If not specified it defaults to 'trace.nettrace'
+
+  --profile
+      A named pre-defined set of provider configurations that allows common tracing scenarios to be specified
+      succinctly. The options are:
+      runtime-basic   Useful for tracking CPU usage and general runtime information. This the default option
+                      if no profile is specified.
+      gc              Tracks allocation and collection performance
+      gc-collect      Tracks GC collection only at very low overhead
+      none            Tracks nothing. Only providers specified by the --providers option will be available.
+
+  --providers <list-of-comma-separated-providers>
+    A list of comma separated EventPipe providers to be enabled.
+    This option adds to the configuration already provided via the --profile argument. If the same provider is configured in both places, this option takes precedence.
+    A provider consists of the name and optionally the keywords, verbosity level, and custom key/value pairs.
+
+    The string is written 'Provider[,Provider]'
+        Provider format: KnownProviderName[:Keywords[:Level][:KeyValueArgs]]
+            KnownProviderName       - The provider's name
+            Keywords                - 8 character hex number bit mask
+            Level                   - A number in the range [0, 5]
+                0 - Always
+                1 - Critical
+                2 - Error
+                3 - Warning
+                4 - Informational
+                5 - Verbose
+            KeyValueArgs            - A semicolon separated list of key=value
+        KeyValueArgs format: '[key1=value1][;key2=value2]'
+            note: values that contain ';' or '=' characters should be surrounded by double quotes ("), e.g., 'key="value;with=symbols";key2=value2'
+
+  --buffersize <Size>
+    Sets the size of the in-memory circular buffer in megabytes. Default 256 MB.
+
+  -f, --format
+    The format of the output trace file.  This defaults to "nettrace" on Windows and "speedscope" on other OSes.
+```
+
+## Collect a trace with `dotnet-trace`
+
+To collect traces, using `dotnet-trace`, you'll need to:
+
+- First, find out the process identifier (pid) of the .NET Core application to collect traces from.
+
+  - On Windows, there are options such as using the task manager or the `tasklist` command on the cmd window.
+  - On Linux, the trivial option could be using `pidof` on the terminal window.
+
+You may also use the command `dotnet-trace list-processes` command to find out what .NET Core processes are running, along with their process IDs.
```

Comment:
```suggestion
You may also use the `dotnet-trace list-processes` command to find out what .NET Core processes are running, along with their PIDs.
```

Reply:
It's also not clear if this info should be indented so it's under the first bullet or if it should be a separate bullet.

![image](https://user-images.githubusercontent.com/12971179/64987662-85931a80-d87e-11e9-9d09-dc351e54bdc6.png)

Reply:
The indent is intentional